packages/amplify-category-analytics/index.js

```diff
@@ -1,11 +1,45 @@
 const path = require('path');
+const inquirer = require('inquirer');
 const pinpointHelper = require('./lib/pinpoint-helper');
+const kinesisHelper = require('./lib/kinesis-helper');
 const { migrate } = require('./provider-utils/awscloudformation/service-walkthroughs/pinpoint-walkthrough');
 
 const category = 'analytics';
 
-function console(context) {
-  pinpointHelper.console(context);
+async function console(context) {
+  const hasKinesisResource = kinesisHelper.hasResource(context);
+  const hasPinpointResporce = pinpointHelper.hasResource(context);
+
+  let selectedResource;
+  if (hasKinesisResource && hasPinpointResporce) {
+    const questions = {
+      name: 'resource',
+      message: 'Select resource',
+      type: 'list',
+      choices: ['kinesis', 'pinpoint'],
+      required: true,
+    };
+
+    const result = await inquirer.prompt(questions);
+    selectedResource = result.resource;
+  } else if (hasKinesisResource) {
+    selectedResource = 'kinesis';
+  } else if (hasPinpointResporce) {
+    selectedResource = 'pinpoint';
+  } else {
+    context.print.error('Neither analytics nor notifications is enabled in the cloud.');
```

Comment:
wording: in the cloud or in the project?

Reply:
this looks correct. `scanCategoryMetaForKinesis` or `scanCategoryMetaForPinpoint` will extract the meta category resource output which is available after the deployment, actually I mimicked the same wording used in pinpoint: [pinpoint-helper.js#L15](https://github.com/ambientlight/amplify-cli/blob/b0af83b299dd9786b9c48ffdc5c86e37f33377fc/packages/amplify-category-analytics/lib/pinpoint-helper.js#L15)